source/client/HandView.py

```diff
@@ -20,16 +31,22 @@ def __init__(self, controller):
         pygame.init()
         # initialize hand_info
         # (hand_info =UICardWrapped elements of current_hand).
+        self.current_hand = []
         self.hand_info = []
-        self.wrapped_hand = []
-        # create window for game - left side is table=PUBLIC, right side is users
-        # TO DO - change this so bottom is current hand and top is table.
-        hand_disp_width = UIC.Disp_Width * UIC.Hand_Col_Fraction
+        # create window for game - top is table=PUBLIC, bottom is users view of hand.
+        # not used: hand_disp_height = UIC.Disp_Width * UIC.Hand_Row_Fraction
         self.display = pygame.display.set_mode((UIC.Disp_Width, UIC.Disp_Height))
         pygame.display.set_caption(self.controller.getName() + " View")
         self.display.fill(UIC.White)
-        # render starting window 
-        self.render
+        self.draw_pile = Cli.ClickableImage\
```

Comment:
I don't like breaking the line like this. Its not THAT long - and breaking it so it looks like (after import fix):
ClickableImage(UIC.Back_Img, \
                           10, \
                           25, \
                           UIC.Back_Img.get_width(), \
                           UIC.Back_Img.get_height(), \
                           0)
is more standard and really readable since all the inputs are in a line.